CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ and this project adheres to
 
 - Extend provisioner to support collections
   [#2830](https://github.com/OpenFn/lightning/issues/2830)
+- Implement collection limiter in the provisioner
```

Comment:
Could we use the word "Add" instead of "Implement" ? Just asking tho, you better than me at naming lol